### PR TITLE
etcd website: added robustness tests meeting information

### PR DESCRIPTION
**Pull request by thedtripp**
Added information about robustness tests meetings with relevant links.

### CODE REVIEW

File: content/en/community/_index.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ main_channels:
       [<i class="fab fa-github"></i>Github Discussions][GD]
     desc: >
       Ask and find answers to your etcd questions.
-community_resources:
+community_meeting_resources:
```
Comment (spzala) [inline content/en/community/_index.md:18]: I don't think you should change this section title.

**Review by spzala — COMMENTED** [review]
@thedtripp thanks for the PR. I have couple of comments that I started adding inline but more think of this update related to meeting, probably a good idea that for all the meetings we just point to this link on the `sig-etcd` page so that we don't need double maintenance and stay latest as per changes on the SIG page, https://github.com/kubernetes/community/blob/master/sig-etcd/README.md#meetings  cc @jmhbnz @ahrtr

File: content/en/community/_index.md [hunk 2]
```diff
@@ -63,7 +63,9 @@ questions:
 You can also chat with other Etcd users and contributors in the #sig-etcd slack
```
Comment (jmhbnz) [inline content/en/community/_index.md:63]: Nit: etcd should always be lowercase, let's fix this while we are updating the page.

```suggestion
You can also chat with other etcd users and contributors in the #sig-etcd slack
```

**Comment by jmhbnz** [comment]
> @thedtripp thanks for the PR. I have couple of comments that I started adding inline but more think of this update related to meeting, probably a good idea that for all the meetings we just point to this link on the `sig-etcd` page so that we don't need double maintenance and stay latest as per changes on the SIG page, https://github.com/kubernetes/community/blob/master/sig-etcd/README.md#meetings cc @jmhbnz @ahrtr

I don't mind the dual update in this case as we don't change meeting details often. I believe having the additional context / details in the website is a better user experience and is more likely to have folks wanting to join versus clicking through to a markdown file with no explanation of what the meetings are really about.

For a long time now https://etcd.io/community has been our single page with all details of how people can interact with our community, would be nice to keep it this way even if it means a little extra pr to keep it in sync every year or so if something changes.

Happy to be overruled if others don't agree.

**Comment by spzala** [comment]
Keeping meeting description is fine, I was suggesting more on pointing to the link to stay up to date on timing in case if anything change (day/hours/frequency) in future and we forget to update time on this page, as it would be bad if someone try to join and couldn't. I don't have a strong opinion to your suggestion and so I am fine either way.

File: content/en/community/_index.md [hunk 3]
```diff
@@ -60,10 +60,12 @@ questions:
 
 {{% cards "main_channels" %}}
 
-You can also chat with other Etcd users and contributors in the #sig-etcd slack
+You can also chat with other etcd users and contributors in the #sig-etcd slack
 channel on [Kubernetes Slack][].
 
-## Community meetings
+## Meeting schedule
+
```
Comment (jmhbnz) [inline content/en/community/_index.md:67]: Suggest adding a note here as suggested by @spzala.
```suggestion

Please refer to [the Kubernetes community](https://github.com/kubernetes/community/blob/master/sig-etcd/README.md#meetings) SIG etcd summary for the latest updates to etcd community meeting schedules.

```

**Review by jmhbnz — COMMENTED** [review]
Adding the link to source of truth seems like a good call, thanks for the clarification @spzala.

**Comment by thedtripp** [comment]
@jmhbnz @spzala I committed the proposed changes, then squashed the commits. Thanks for the feedback.

**Comment by spzala** [comment]
> Adding the link to source of truth seems like a good call, thanks for the clarification @spzala.

Thanks @jmhbnz and @thedtripp !!

**Review by spzala — APPROVED** [review]

**Pull request merged**